src/vma/dev/ib_ctx_handler.h

```diff
@@ -45,52 +45,45 @@ class ib_ctx_handler : public event_handler_ibverbs
 public:
 	ib_ctx_handler(struct ibv_context* ctx, ts_conversion_mode_t ctx_time_converter_mode);
 	virtual ~ib_ctx_handler();
+
 	/*
 	 * on init or constructor:
 	 *      register to event manager with m_channel and this.
 	 * */
-	//void execute(struct ibv_async_event ibv_event) { handle_ibv_event(ibv_event); }
-	void                    set_dev_configuration();
-	ibv_mr*                 mem_reg(void *addr, size_t length, uint64_t access);
-	void                    mem_dereg(ibv_mr *mr);
-	ibv_port_state          get_port_state(int port_num);
+	ibv_pd*                 get_ibv_pd() { return m_p_ibv_pd;}
 	ibv_device*             get_ibv_device() { return m_p_ibv_device;}
-	ibv_pd*			get_ibv_pd() { return m_p_ibv_pd;}
 	struct ibv_context*     get_ibv_context() { return m_p_ibv_context;}
-	vma_ibv_device_attr&    get_ibv_device_attr() { return m_ibv_device_attr;}
-	struct ibv_port_attr    get_ibv_port_attr(int port_num);
+	vma_ibv_device_attr*    get_ibv_device_attr() { return m_p_ibv_device_attr;}
+	ibv_mr*                 mem_reg(void *addr, size_t length, uint64_t access);
+	void                    mem_dereg(ibv_mr *mr);
 	bool                    is_removed() { return m_removed;}
-	virtual void            handle_event_ibverbs_cb(void *ev_data, void *ctx);
-	void                    handle_event_DEVICE_FATAL();
 	ts_conversion_mode_t    get_ctx_time_converter_status();
 	void                    set_flow_tag_capability(bool flow_tag_capability); 
 	bool                    get_flow_tag_capability() { return m_flow_tag_enabled;} // m_flow_tag_capability
-
+	ibv_port_state          get_port_state(int port_num);
+	virtual void            handle_event_ibverbs_cb(void *ev_data, void *ctx);
 	inline void convert_hw_time_to_system_time(uint64_t hwtime, struct timespec* systime)
 	{
 		m_p_ctx_time_converter->convert_hw_time_to_system_time(hwtime, systime);
 	}
-
 private:
-	struct ibv_context*     m_p_ibv_context;
-	struct ibv_port_attr    m_ibv_port_attr;
+	void                    handle_event_DEVICE_FATAL();
```

Comment:
@rafiw probably `handle_event_DEVICE_FATAL()` could be replaced with `handle_event_device_fatal` to improve coding style in this pull request too.